src/goto-programs/rebuild_goto_start_function.cpp

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************\
+ Module: C Test Case Generator
```

Comment:
Is that really the appropriate module? It doesn't seem to be an accurate description now.